src/main/kotlin/no/nav/familie/ef/sak/behandling/RevurderingService.kt

```diff
@@ -152,7 +153,7 @@ class RevurderingService(
         val sisteBehandling = behandlingService.hentBehandlinger(revurdering.fagsakId)
             .filter { it.id != revurdering.id }
             .filter { it.resultat != BehandlingResultat.HENLAGT }
-            .maxByOrNull { it.sporbar.opprettetTid }
+            .sisteFerdigstilteBehandling()
```

Comment:
Manuell revurdering; tar høyde for avslag og henter vilkår basert på dette, MEN; 
- journalføring av ettersending/papir som oppretter ny behandling -> der tar vi ikke høyde for dette?


Bør denne funksjonen bli "felles" og dermed også brukes av jfr som oppretter ny behandling UTEN strukturert søknadsdata?

(Automatisk journalføring og søknadsbasert journalføring nuller jo ut vilkår så det er ikke noe stress)

Reply:
I RevurderingService så brukes denne "forrigeBehandlingId" for å kopiere barn og vilkår
Mens `forrigeBehandlingId` på behandlingen settes likt for både revurdering og journalføring, som alltid blir det siste ferdigstilte behandlingen

Men tror du kanskje tenker på noe annet? 👀